app/src/main/java/ch/epfl/sdp/MainActivity.kt

```diff
@@ -6,16 +6,27 @@ import androidx.appcompat.app.AppCompatActivity
 import ch.epfl.sdp.databinding.ActivityMainBinding
 import ch.epfl.sdp.lobby.LobbyActivity
 import ch.epfl.sdp.authentication.LoginActivity
+import ch.epfl.sdp.db.IRepoFactory
+import ch.epfl.sdp.db.MockDB
+import ch.epfl.sdp.lobby.global.GlobalLobbyActivity
+import ch.epfl.sdp.lobby.global.IGlobalLobbyRepository
+import ch.epfl.sdp.lobby.global.MockGlobalLobbyRepository
 
 class MainActivity : AppCompatActivity() {
     private lateinit var binding: ActivityMainBinding
+    private val repositoryFactory = object : IRepoFactory {
+        override fun makeGlobalLobbyRepository(): IGlobalLobbyRepository {
+            return MockGlobalLobbyRepository()
+        }
+    }
```

Comment:
Pour y retrouver facilement plus tard et ne pas oublier
```suggestion
    // TODO Inject repository with dagger or get from Application context
    private val repositoryFactory = object : IRepoFactory {
        override fun makeGlobalLobbyRepository(): IGlobalLobbyRepository {
            return MockGlobalLobbyRepository()
        }
    }
```